datacommons/core.py

```diff
@@ -0,0 +1,130 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" Data Commons base Python Client API.
+
+Contains wrapper functions for get_property_labels, get_property_values, and
+get_triples
+"""
+
+from collections import defaultdict
+
+import pandas as pd
+
+import datacommons.utils as utils
+import requests
+
+# ----------------------------- WRAPPER FUNCTIONS -----------------------------
+
+def get_property_labels(dcid, outgoing=True, reload=False):
```

Comment:
I guess the assumption is that a node will never have too many properties. For now that seems more than reasonable, but stands out because almost all other methods have a limit on return length.